.github/workflows/wheels.yml

```diff
@@ -0,0 +1,80 @@
+name: cuDF wheels
+
+on:
+  push:
+    branches:
+      - 'pull-request/[0-9]+'
```

Comment:
Like with last time, noting that we should clean up the `pull-request` trigger before merging.